added myself

nyborrobyn opened the pull request.

bigethan commented:
Hey Robyn, your PR is conflicted (likely due to merging someone else's name).  Could you pull down the latest version of the repo, fix any conflicts, and push back up to this branch?

zestyping commented:
I've fixed the merge conflict and added you.  Thanks for signing!

nyborrobyn commented:
Thanks, all. Wasn't sure I did it right, so thank you!




Robyn Perry
@nyborrobyn
Aikuma Project <http://www.aikuma.org/>
*UC Berkeley School of Information MIMS, 2015*

On Tue, Dec 13, 2016 at 9:46 PM, Ka-Ping Yee <notifications@github.com>
wrote:

> I've fixed the merge conflict and added you. Thanks for signing!
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/neveragaindottech/neveragaindottech.github.io/pull/363#issuecomment-266948509>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AFGjb2Qk2xOb3XuXY30cdBPjke4hGyBSks5rH4LNgaJpZM4LMhQm>
> .
>
